src/app.js

```diff
@@ -4,27 +4,32 @@ import morgan from "morgan";
 import swaggerUI from 'swagger-ui-express';
 import docs from'./documentation';
 import { sequelize } from "./database/models";
-import router from "./routes/welcome.router";
+import register from "./routes/router.js";
 
 dotenv.config();
 
 const app = express();
 
 export const connectDB = async () => {
-	try {
-		await sequelize.authenticate();
-		console.log('Database connection established successfully');
-	} catch (err) {
-		console.log(`Database connection failed: ${err}`);
-		process.exit(1);
-	}
+  try {
+    await sequelize.authenticate();
+    console.log("Database connection established successfully");
+  } catch (err) {
+    console.log(`Database connection failed: ${err}`);
+    process.exit(1);
+  }
 };
 
 app.use(morgan('dev'));
 app.use(express.json());
 app.use(express.urlencoded({ extended: false }));
-
-app.use('/', router)
 app.use('/api-docs',swaggerUI.serve,swaggerUI.setup(docs));
+app.use("/", register);
+
+app.use((req, res) =>
+  res.status(400).json({
+    Error: "Unavailable Request",
+  })
+);
```

Comment:
What's the role of this middleware?

Reply:
It's for catching any Errors regarding the user accessing unprovided endpoints.